PrismaDB-QueryAST/DDL/AlterTableQuery.cs

```diff
@@ -27,16 +27,16 @@ public AlterTableQuery(AlterType type, string tableName)
         public AlterTableQuery(AlterType type, TableRef table)
         {
             AlterType = type;
-            TableName = table.Clone();
+            TableName = table;
             AlteredColumns = new List<AlteredColumn>();
         }
 
         public AlterTableQuery(AlterTableQuery other)
         {
-            TableName = other.TableName.Clone();
+            TableName = other.TableName;
```

Comment:
As mentioned, in constructors that take another of the same object, it should clone.

Reply:
ok noted, will make the necessary changes